doc/api/http.markdown

```diff
@@ -103,7 +103,7 @@ of these values set to their respective defaults.
 To configure any of them, you must create your own [`http.Agent`][] object.
 
 ```javascript
-var http = require('http');
+const http = require('http');
```

Comment:
It might actually be better to have huge docs only churn in a single commit, instead of [32 commits](https://github.com/nodejs/node/pull/3662). People are a lot less likely to bisect and blame the docs.